extensions/ql-vscode/src/common/vscode/commands.ts

```diff
@@ -0,0 +1,25 @@
+import { commands } from "vscode";
+import { commandRunner } from "../../commandRunner";
+import { CommandFunction, CommandManager } from "../../packages/commands";
+
+/**
+ * Intializes a command manager for VSCode, wrapping the commandRunner
+ * and vscode.executeCommand.
+ */
+export function initializeVSCodeCommandManager<
```

Comment:
(Minor and optional) `initialize` makes me think that there is some work to be done to initialize something, but I think that here we're just creating an object. Would `createVSCodeCommandManager` be a better name?